templates/codelists/version.html

```diff
@@ -211,6 +211,7 @@ <h6>Versions</h6>
         {% elif version.is_draft %}
         <span class="badge badge-primary">Draft</span>
         {% endif %}
+        <span>(updated {{ version.updated_at | date:'Y-m-d' }})
```

Comment:
Unless I'm missing something, maybe this needs the closing tag?

```suggestion
        <span>(updated {{ version.updated_at | date:'Y-m-d' }})</span>
```

Reply:
Oh wow, don't know how I missed that one!